src/server/utilities/liteATIClickTracking/index.ts

```diff
@@ -0,0 +1,94 @@
+export default function trackingScript() {
```

Comment:
Can we use fat arrow syntax here, or is it not compatible with older browsers?
```suggestion
export default () =>  {
```

Reply:
Can we rename this (and all references) to match the filename?
```suggestion
export default function liteATIClickTracking() {
```